### PR TITLE
Fix configuration files

### PR DESCRIPTION
**Pull request by mauroservienti**

### CODE REVIEW

File: ASP.Net Mvc Core/Divergent.Shipping.API.Host/App.config [hunk 1]
```diff
@@ -7,28 +7,10 @@
...
-        <assemblyIdentity name="Microsoft.Owin" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-3.1.0.0" newVersion="3.1.0.0" />
-      </dependentAssembly>
-      <dependentAssembly>
-        <assemblyIdentity name="System.Web.Cors" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-5.2.3.0" newVersion="5.2.3.0" />
-      </dependentAssembly>
-    </assemblyBinding>
-  </runtime>
+
```
Comment (mauroservienti) [inline ASP.Net Mvc Core/Divergent.Shipping.API.Host/App.config]:
```suggestion
```

File: ASP.Net Mvc Core/Divergent.Sales.API.Host/App.config [hunk 2]
```diff
@@ -7,28 +7,10 @@
...
-        <assemblyIdentity name="Microsoft.Owin" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-3.1.0.0" newVersion="3.1.0.0" />
-      </dependentAssembly>
-      <dependentAssembly>
-        <assemblyIdentity name="System.Web.Cors" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-5.2.3.0" newVersion="5.2.3.0" />
-      </dependentAssembly>
-    </assemblyBinding>
-  </runtime>
+
```
Comment (mauroservienti) [inline ASP.Net Mvc Core/Divergent.Sales.API.Host/App.config]:
```suggestion
```

File: ASP.Net Mvc Core/Divergent.Shipping.Data/App.config [hunk 3]
```diff
@@ -11,9 +11,6 @@
       <provider invariantName="System.Data.SQLite.EF6" type="System.Data.SQLite.EF6.SQLiteProviderServices, System.Data.SQLite.EF6" />
     </providers>
   </entityFramework>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
-  </startup>
```
Comment (mauroservienti) [inline ASP.Net Mvc Core/Divergent.Shipping.Data/App.config]:
```suggestion
```

File: ASP.Net Mvc Core/Divergent.Sales.Data/App.config [hunk 4]
```diff
@@ -11,9 +11,6 @@
       <provider invariantName="System.Data.SQLite.EF6" type="System.Data.SQLite.EF6.SQLiteProviderServices, System.Data.SQLite.EF6" />
     </providers>
   </entityFramework>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
-  </startup>
```
Comment (mauroservienti) [inline ASP.Net Mvc Core/Divergent.Sales.Data/App.config]:
```suggestion
```

File: ASP.Net Mvc Core UI Composition/Divergent.Shipping.Data/App.config [hunk 5]
```diff
@@ -11,9 +11,6 @@
       <provider invariantName="System.Data.SQLite.EF6" type="System.Data.SQLite.EF6.SQLiteProviderServices, System.Data.SQLite.EF6" />
     </providers>
   </entityFramework>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
-  </startup>
```
Comment (mauroservienti) [inline ASP.Net Mvc Core UI Composition/Divergent.Shipping.Data/App.config]:
```suggestion
```

File: ASP.Net Mvc Core UI Composition/Divergent.Sales.Data/App.config [hunk 6]
```diff
@@ -11,9 +11,6 @@
       <provider invariantName="System.Data.SQLite.EF6" type="System.Data.SQLite.EF6.SQLiteProviderServices, System.Data.SQLite.EF6" />
     </providers>
   </entityFramework>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
-  </startup>
```
Comment (mauroservienti) [inline ASP.Net Mvc Core UI Composition/Divergent.Sales.Data/App.config]:
```suggestion
```

File: ASP.Net Core API Gateway - 02/Divergent.Shipping.Data/App.config [hunk 7]
```diff
@@ -11,9 +11,6 @@
       <provider invariantName="System.Data.SQLite.EF6" type="System.Data.SQLite.EF6.SQLiteProviderServices, System.Data.SQLite.EF6" />
     </providers>
   </entityFramework>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
-  </startup>
```
Comment (mauroservienti) [inline ASP.Net Core API Gateway - 02/Divergent.Shipping.Data/App.config]:
```suggestion
```

File: ASP.Net Core API Gateway - 02/Divergent.Sales.Data/App.config [hunk 8]
```diff
@@ -11,9 +11,6 @@
       <provider invariantName="System.Data.SQLite.EF6" type="System.Data.SQLite.EF6.SQLiteProviderServices, System.Data.SQLite.EF6" />
     </providers>
   </entityFramework>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
-  </startup>
```
Comment (mauroservienti) [inline ASP.Net Core API Gateway - 02/Divergent.Sales.Data/App.config]:
```suggestion
```

File: ASP.Net Core API Gateway - 01/Divergent.Shipping.Data/App.config [hunk 9]
```diff
@@ -11,9 +11,6 @@
       <provider invariantName="System.Data.SQLite.EF6" type="System.Data.SQLite.EF6.SQLiteProviderServices, System.Data.SQLite.EF6" />
     </providers>
   </entityFramework>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
-  </startup>
```
Comment (mauroservienti) [inline ASP.Net Core API Gateway - 01/Divergent.Shipping.Data/App.config]:
```suggestion
```

File: ASP.Net Core API Gateway - 01/Divergent.Sales.Data/App.config [hunk 10]
```diff
@@ -11,9 +11,6 @@
       <provider invariantName="System.Data.SQLite.EF6" type="System.Data.SQLite.EF6.SQLiteProviderServices, System.Data.SQLite.EF6" />
     </providers>
   </entityFramework>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
-  </startup>
```
Comment (mauroservienti) [inline ASP.Net Core API Gateway - 01/Divergent.Sales.Data/App.config]:
```suggestion
```

File: ASP.Net Mvc Core UI Composition/Divergent.Shipping.API.Host/App.config [hunk 11]
```diff
@@ -7,28 +7,10 @@
...
-        <assemblyIdentity name="Microsoft.Owin" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-3.1.0.0" newVersion="3.1.0.0" />
-      </dependentAssembly>
-      <dependentAssembly>
-        <assemblyIdentity name="System.Web.Cors" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-5.2.3.0" newVersion="5.2.3.0" />
-      </dependentAssembly>
-    </assemblyBinding>
-  </runtime>
+
```
Comment (mauroservienti) [inline ASP.Net Mvc Core UI Composition/Divergent.Shipping.API.Host/App.config]:
```suggestion
```

File: ASP.Net Mvc Core UI Composition/Divergent.Sales.API.Host/App.config [hunk 12]
```diff
@@ -7,28 +7,10 @@
...
-        <assemblyIdentity name="Microsoft.Owin" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-3.1.0.0" newVersion="3.1.0.0" />
-      </dependentAssembly>
-      <dependentAssembly>
-        <assemblyIdentity name="System.Web.Cors" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-5.2.3.0" newVersion="5.2.3.0" />
-      </dependentAssembly>
-    </assemblyBinding>
-  </runtime>
+
```
Comment (mauroservienti) [inline ASP.Net Mvc Core UI Composition/Divergent.Sales.API.Host/App.config]:
```suggestion
```

**Comment by mauroservienti** [comment]
@wip ready for review

**Comment by gr2m** [comment]
Note that the `@wip ready for review` has to be added to the pull request body, WIP is not looking for it in comments, it would require another request. We might change that in future when we switch to GraphQL though :) 

Can you please edit the pull request title to trigger another pull_request event? I see several failed events for this repository due to the DNS problems I mentioned

**Pull request merged**